website/content/api-docs/system/remount.mdx

```diff
@@ -6,16 +6,32 @@ description: >-
   endpoint.
 ---
 
-# `/sys/remount`
+# Remount ( API )
 
-The `/sys/remount` endpoint is used remount a mounted backend to a new endpoint.
+The Remount documentation details the endpoints required to trigger and monitor the status of a remount operation.
 
 ## Move Backend
 
-This endpoint moves an already-mounted backend to a new mount point. 
+The `/sys/remount` endpoint moves an already-mounted backend to a new mount point. This works for both secret engines
+and auth methods.
+
+The endpoint can be used to move the backend across namespaces as well, with the namespaces supplied as prefixes
```

Comment:
```suggestion
The endpoint may be used to move the backend across namespaces as well, with the namespaces supplied as prefixes
```

Reply:
I think `can` is correct here imo, since the vault operator decides how to use this endpoint